docs/toc.md

```diff
@@ -26,6 +26,7 @@
     * [Committee Scheduler](consensus/scheduler.md)
     * [Root Hash](consensus/roothash.md)
     * [Key Manager](consensus/keymanager.md)
+  * [Genesis Document](consensus/genesis.md)
```

Comment:
Also update `index.md`? We could also remove the TOC from index and mention that TOC is in the sidebar.

Reply:
Yea, I missed that. TOC in index.md is probably useful for browsing docs directly on GitHub, but it is prone to someone forgetting to update it :slightly_smiling_face:.